nixos/modules/services/misc/ollama.nix

```
@@ -1,71 +1,72 @@
 { config, lib, pkgs, ... }:
 let
-  inherit (lib) types mkBefore;
+  inherit (lib) literalExpression types mkBefore;
 
   cfg = config.services.ollama;
   ollamaPackage = cfg.package.override {
     inherit (cfg) acceleration;
   };
+
+  staticUser = cfg.user != null && cfg.group != null;
 in
 {
   imports = [
     (lib.mkRemovedOptionModule [ "services" "ollama" "listenAddress" ]
       "Use `services.ollama.host` and `services.ollama.port` instead.")
+    (lib.mkRemovedOptionModule [ "services" "ollama" "sandbox" ]
+      "Set `services.ollama.user` and `services.ollama.group` instead.")
+    (lib.mkRemovedOptionModule [ "services" "ollama" "writablePaths" ]
+      "The models directory is now always writable." )
```

```suggestion
    (lib.mkRemovedOptionModule [ "services" "ollama" "writablePaths" ]
      "The `models` directory is now always writable. To make other directories writable, use `systemd.services.ollama.serviceConfig.ReadWritePaths`." )
```
I think this should mention `systemd.services.ollama.serviceConfig.ReadWritePaths` for anyone that really does need to whitelist any directories other than `models`.